core/field_dropdown.js

```diff
@@ -239,6 +239,10 @@ Blockly.FieldDropdown.prototype.positionMenu_ = function(menu) {
   this.createWidget_(menu);
   var menuSize = Blockly.utils.uiMenu.getSize(menu);
 
+  if (menuSize.height > 300) {
```

Comment:
Make this a constant (see https://github.com/google/blockly/pull/1483/files#diff-c7c9714a61e86d3fe5334619514cea6cL69 for an example) and add a comment that it's also referenced in css.js as part of `.blocklyDropdownMenu `.

Reply:
Done!